geonode_mapstore_client/templates/geonode-mapstore-client/_geonode_config.html

```diff
@@ -88,7 +89,9 @@
                     defaultLayerFormat: defaultLayerFormat,
                     timeEnabled: timeEnabled,
                     allowedDocumentTypes: allowedDocumentTypes,
-                    isMobile: isMobile
+                    isMobile: isMobile,
+                    dataset_upload_size: defaultMaxUploadSize,
+                    document_upload_size: defaultMaxUploadSize,
```

Comment:
please use camelCase for variables in js
```suggestion
                    datasetMaxUploadSize: defaultMaxUploadSize,
                    documentMaxUploadSize: defaultMaxUploadSize,
```

Reply:
updated